mixer/cmd/mixc/cmd/root.go

```diff
@@ -69,6 +78,12 @@ func addAttributeFlags(cmd *cobra.Command, rootArgs *rootArgs) {
 		"Address and port of a running Mixer instance")
 	cmd.PersistentFlags().IntVarP(&rootArgs.repeat, "repeat", "r", 1,
 		"Sends the specified number of requests in quick succession")
+	cmd.PersistentFlags().IntVarP(&rootArgs.concurrency, "concurrency", "c", 1,
```

Comment:
Let's not use the word "goroutine" in the command-line printed to the user, here and elsewhere. Talk in terms of concurrency, not goroutines.

Thanks.

Reply:
Reworded to avoid goroutine. Thanks!